master/reference/node/configuration.md

```diff
@@ -24,7 +24,7 @@ The `calico/node` container is primarily configured through environment variable
 | CALICO_NETWORKING_BACKEND | Describes which BGP networking backend to use| gobgp, bird, none | bird |
 | CALICO_IPV4POOL_CIDR | The IPv4 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. | IPv4 CIDR | 192.168.0.0/16 | |
 | CALICO_IPV6POOL_CIDR | The IPv6 Pool to create if none exists at start up. It is invalid to define this variable and NO_DEFAULT_POOLS. | IPv6 CIDR | fd80:24e2:f998:72d6::/64 | |
-| CALICO_IPIP_ENABLED  | Enable IPIP on any IP Pools created at start up. | string | false | |
+| CALICO_IPV4POOL_IPIP | IPIP Mode to use on an IPv4 POOL being created at start up | off, always, cross-subnet | off | |
```

Comment:
Wording seems a little odd.

How about "IPIP Mode to use for the IPv4 IP Pool created at startup."